Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM oraclelinux:8.9
+
+# Download packages to prep container
+RUN    chmod +t /tmp /var/tmp && \
+    update-ca-trust && \
+    update-ca-trust force-enable && \
+    dnf install -y openssl curl python3.11 git && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf/ /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
+
+# Disable ChaCha20 Algorithms
+RUN set -eux; \
+    disabledAlgorithms=' \
+       TLS_CHACHA20_POLY1305_SHA256, \
+       TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256, \
+       TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256, \
+       TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256, \
+    '; \
+    disabledAlgorithms="${disabledAlgorithms//[[:space:]]/}"; \
+    sed -i "s/^jdk\.tls\.disabledAlgorithms=/jdk.tls.disabledAlgorithms=$disabledAlgorithms/" \
+       /usr/share/crypto-policies/FIPS/java.txt;
+
+# Set environment variables to ensure Python output is not buffered
+ENV PYTHONUNBUFFERED=1
+
+# Install Poetry by downloading the installation script and executing it
+RUN curl -sSL https://install.python-poetry.org | python3 -
+
+# Add Poetry to the system path
+ENV PATH="/root/.local/bin:${PATH}"
+
+# Change dir to pull gafm 
+WORKDIR /usr/share
+
+# Pull GAFM
+RUN git clone https://github.com/mssalvatore/gafm.git 
```

Comment:
I think this should pull from the local files. Otherwise you can't test your changes.

Reply:
many users I encounter cannot clone git repos, but are able to use git within a dockerfile.
Providing one dockerfile instead of requiring a git clone is more concise imo

Reply:
Is there any way to add a conditional based on an environment variable or CLI parameter?